### ISSUE TITLE
git submodules are not correctly sets

### ISSUE THREAD

**Issue by neomilium** [haum/lightstep#2]
These commands should be enough to fetch submodules, but does not:

```
git submodule init
git submodule update
```

**Comment by matael** [comment]
will fix soon. A workaround is : 

```
mkdir -p lightstep_fw/_submodules/arduino-libs
cd lightstep_fw/_submodules
git clone https://github.com/sudar/Arduino-Makefile.git arduino-mk
cd arduino-libs
git clone https://github.com/haum/polychrhaum PolychrHAUM
git clone https://github.com/FastLED/FastLED.git
cd PolychrHAUM
git checkout full_fastled_support
```

**Comment by jblb** [comment]
normalement c'est corrigé

**Comment by matael** [comment]
Fix confirmed in commit 3373e006de . Thank you @jblb
